### ISSUE TITLE
Gopack import path incorrect for Svelte store

### ISSUE THREAD

**Issue by jimafisk** [plentico/plenti#184]
Go pack is turning the import at the top of `node_modules/svelte/store/index.mjs`:

```
import { noop, safe_not_equal, subscribe, run_all, is_function } from '../internal';
export { get_store_value as get } from '../internal';
```

Into the following in `public/spa/web_modules/svelte/store/index.mjs`:

```
import { noop, safe_not_equal, subscribe, run_all, is_function } from '../../internal/index.mjs';
export { get_store_value as get } from '../../internal/index.mjs';
```

This causes Chrome to throw:
```
GET http://localhost:3000/spa/web_modules/internal/index.mjs net::ERR_ABORTED 404 (Not Found)
```

Because the actual path needs to be `http://localhost:3000/spa/web_modules/svelte/internal/index.mjs` (it's missing the "svelte" dir). There's one too many `..` in the beginning of the converted path.

**Comment by jimafisk** [comment]
```go
package main

import (
	"fmt"
	"path"
	"strings"
)

func main() {
	pathStr := "../internal"
	entryPoint := "./index.mjs"
	fmt.Println(strings.Split(pathStr, "/")[0])
	fmt.Println(path.Join(pathStr, entryPoint))

	pathStr = "./internal"
	entryPoint = "./index.mjs"
	fmt.Println(strings.Split(pathStr, "/")[0])
	fmt.Println(path.Join(pathStr, entryPoint))
}
```
Prints:
- `..`
- `../internal/index.mjs`
and
- `.`
- `internal/index.mjs`

The problem here is the different treatment of `path.Join` when goin up a directory `..` vs the current directory `.`

**Comment by jimafisk** [comment]
Something like this should account for a single `.` or double `..`:

```go
fixCurrentDir := ""
if strings.Split(pathStr, "/")[0] == "." {
	fixCurrentDir = strings.Split(pathStr, "/")[0] + "/"
}
fmt.Println(fixCurrentDir + path.Join(pathStr, entryPoint))
```

**Comment by jimafisk** [comment]
Fixed in [v0.5.2](https://github.com/plentico/plenti/releases/tag/v0.5.2)
